### ISSUE TITLE
Make the live notebook element more prominent

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-examples#94]
When I've shown people examples.dask.org they often don't realize that they can click on the "Launch Binder" button and get a live session.  This is despite our header at the top which says:

> You can run this notebook in a [live session]() or view it [on  Github]()

I think that we might make this more prominent by 

1.  Using a button, similar to the "Launch Binder" button, but more obvious to people who are unfamiliar with Binder
2.   Making that button very large?
3.  Making that button stay on the screen, even after the user scrolls down?
4.  ...?

If only we knew someone with some basic web design skills ... 

cc @jrbourbeau , in case you or anyone around you has ideas ;)

**Comment by jrbourbeau** [comment]
Do you think it's more an issue of seeing the header, not knowing what binder is, and so ignoring and moving on. Or not noticing the header at all?

If we think the header isn't standing out enough, one simple (and hopefully effective) thing we could do is use the sphinx `.. note::` directive to place the header in a blue box that should draw more attention to it (e.g. I've used this in another project at https://jrbourbeau.github.io/pyunfold/notebooks/tutorial)

**Comment by mrocklin** [comment]
Yeah, I think that the note directive would be great.  I also think that we
could improve upon the binder button (See
https://github.com/jupyterhub/mybinder.org-deploy/issues/1147 ) .  I think
that any change in any of these directions would have a large impact.

**Comment by jrbourbeau** [comment]
I opened up #95 which has some updates (happy to add more as we see fit). Glad to see the upstream discussion : )
